tools/llvm-split/llvm-split.cpp

```diff
@@ -40,6 +41,14 @@ static cl::opt<bool>
     PreserveLocals("preserve-locals", cl::Prefix, cl::init(false),
                    cl::desc("Split without externalizing locals"));
 
+static cl::opt<bool>
+    SplitHCCKernels("split-hcc-kernels", cl::Prefix, cl::init(false),
+                   cl::desc("Split HCC Kernels and print number of files"));
+
+static cl::opt<unsigned>
+    SplitHCCThreshold("split-hcc-threshold", cl::Prefix, cl::init(100),
+                   cl::desc("Split HCC Kernels if above kernel count threshold"));
```

Comment:
@scchan this was originally for ThinLTO and kernel splitting. Do we still need this?